lxd/storage_shared.go

```diff
@@ -23,6 +24,11 @@ type storageShared struct {
 	volume *api.StorageVolume
 }
 
+// OS returns the sys.OS facade instance used by this storage.
+func (s *storageShared) OS() *sys.OS {
+	return s.s.OS
+}
```

Comment:
Same argument as for OS() in container_lxc.go, this is making it harder to figure out what the code does by requiring the user go read that one line function to be sure of the behavior.

In this case, it's not even making things shorter "s.s.OS" vs "s.OS()".

Reply:
Ok, I dropped it in 17615b945089b5a35a6802f0b8064ca1466e9291.